jenkins/ci.suse.de/templates/cloud-mkcloud-job-sap-ha-template.yaml

```diff
@@ -0,0 +1,28 @@
+- job-template:
+    name: 'cloud-mkcloud{version}-job-sap-ha-{arch}'
+    node: cloud-trigger
+    disabled: '{obj:disabled}'
+
+    triggers:
+      - timed: 'H H * * *'
+
+    logrotate:
+      numToKeep: -1
+      daysToKeep: 7
+
+    builders:
+      - trigger-builds:
+        - project: openstack-mkcloud
+          condition: SUCCESS
+          block: true
+          current-parameters: true
+          predefined-parameters: |
+            TESTHEAD=1
+            cloudsource=develcloud{version}
+            nodenumber=10
+            clusterconfig=data=2:services=3:network=3
+            storage_method=swift
+            hacloud=1
```

Comment:
At the moment we don't use bonding for the mkcloud runs, this might change in the future but for now I would like to keep it as it is.